packages/components/src/styles/components/tabs.scss

```diff
@@ -9,6 +9,36 @@
 
 @use "../mixins/focus-ring" as *;
 
+// Variants:
+
+// medium = default
+.hds-tabs--size-medium {
+  .hds-tabs__tab {
+    height: var(--token-tabs-tab-height);
+    padding: var(--token-tabs-tab-padding-vertical) var(--token-tabs-tab-padding-horizontal);
+  }
+
+  .hds-tabs__tab-button {
+    font-size: var(--token-typography-body-200-font-size);
+  }
+}
+
+.hds-tabs--size-large {
+  --token-tabs-tab-height: 48px; // override the default value
```

Comment:
Let me know if overriding the tokens like this or if I should create new tokens.

Reply:
probably better to create new tokens (eg. `token-tabs-tab-height-[medium|large]`, `--token-tabs-tab-padding-[horizontal|vertical]-[medium|large]` and delete the old ones
this will means that we also need to release the tokens package as `major`, but it should not be a big deal, it's used only in a single place (in the `web` app for Developer: https://github.com/search?q=token-tabs-tab-height+user%3Ahashicorp&type=code&ref=advsearch) and it should be easy for them to replace